docs/components.js

```diff
@@ -0,0 +1,137 @@
+import Calendar from '../calendar/index.vue';
+import DatePicker from '../datepicker/index.vue';
+
+export default {
+  calendar: {
+    component: Calendar,
+    title: 'Calendar',
+    description: 'The Calendar component provides a responsive, dynamic, accessible, localized month-based calendar.',
+    props: [
```

Comment:
I'm curious if there's a way to make this work from JSDoc or automatically generated from the component itself; but that doesn't need to be addressed in this PR. A thought for another time, recorded so I don't forget it.

Reply:
Actually, some Vue component projects use additional fields in the components props section like `description` so source code is self-describing. But I didn't choose this approach as it adds that description to the final bundle which is not something ideal. 

For JSDocs I will be happy helping to get it working with next PRs.

Reply:
Yah, absolutely not with this PR, but we can totally discuss a way of automating this in an issue for another PR